### ISSUE TITLE
RCS1135: "Declare enum mem w/ zero value" Should Take into Account 1 << 0

### ISSUE THREAD

**Issue by zacharylayne** [JosefPihrt/Roslynator#934]
**Steps to Reproduce**:
1. Create flags enum.
2. Add entry with value explicitly set to 1 << 0

```csharp
[Flags]
enum Foo
{
    None = 1 << 0
}
```

**Actual Behavior**:

`RCS1135: Declare enum member with zero value (when enum has FlagsAttribute)` in **Error List**

**Expected Behavior**:

RCS1135 is not triggered.

**Comment by zacharylayne** [comment]
It's 3:00 AM. I had a durr and just realized as soon as I submitted the issue. :P
